mne/preprocessing/ica.py

```diff
@@ -222,23 +222,27 @@ class ICA(ContainsMixin):
         Channel names resulting from initial picking.
     n_components_ : int
         If fit, the actual number of PCA components used for ICA decomposition.
-    pre_whitener_ : ndarray, shape (n_channels, 1) or (n_channels, n_channels)
+    explained_variance_ratio_ : array, shape (n_components_,) | None
+        If fit, the relative variance explained by each ICA component. If not
+        fit, ``None``.
+    pre_whitener_ : array, shape (n_channels, 1) or (n_channels, n_channels)
         If fit, array used to pre-whiten the data prior to PCA.
-    pca_components_ : ndarray, shape ``(n_channels, n_channels)``
+    pca_components_ : array, shape (n_channels, n_channels)
         If fit, the PCA components.
-    pca_mean_ : ndarray, shape (n_channels,)
+    pca_mean_ : array, shape (n_channels,)
         If fit, the mean vector used to center the data before doing the PCA.
-    pca_explained_variance_ : ndarray, shape ``(n_channels,)``
-        If fit, the variance explained by each PCA component.
-    mixing_matrix_ : ndarray, shape ``(n_components_, n_components_)``
+    pca_explained_variance_ : array, shape (n_channels,)
+        If fit, the variance explained by each PCA component. Note that the
+        variances are not unitized, i.e., don't sum to 1.
+    mixing_matrix_ : array, shape (n_components_, n_components_)
         If fit, the whitened mixing matrix to go back from ICA space to PCA
         space.
         It is, in combination with the ``pca_components_``, used by
         :meth:`ICA.apply` and :meth:`ICA.get_components` to re-mix/project
         a subset of the ICA components into the observed channel space.
         The former method also removes the pre-whitening (z-scaling) and the
         de-meaning.
-    unmixing_matrix_ : ndarray, shape ``(n_components_, n_components_)``
+    unmixing_matrix_ : array, shape (n_components_, n_components_)
```

Comment:
A trailing underscore in RST is a link. By taking these out of code, that's how it's interpreted now. Put these back in double-backticks and you should be good. (Or you can maybe escape the trailing `_`, not sure -- but it doesn't seem worth it.)

Reply:
haha, oops, my fault!  I thought that was a mistake

Reply:
Oh how I loathe RST.......... 🤬